### DISCUSSION TITLE
有查询树脂或者是树脂推送的插件吗

### DISCUSSION THREAD

**Discussion by LddsnHong** [Arondight/Adachi-BOT#789] [Q&A] [answered]
有查询树脂或者是树脂推送的插件吗

**Comment by mark9804** [comment, marked as answer]
https://github.com/ixCiel/Adachi-BOT

**Reply by CeciliaAki** [reply]
你好，我是一个计算机小白，只学过C，想请问一下这个链接点进去之后是一个新的bot，要怎样才能完成插件安装呢？问题或略显笨，百忙之中叨扰多有抱歉，希望能够获得解答。><

**Reply by Arondight** [reply]
不是插件，是这个项目的替代品

**Reply by CeciliaAki** [reply]
好的，感谢

**Comment by mark9804** [comment]
这个bot和我们的其他功能基本上是一样的，直接用他的bot就可以了

**Reply by CeciliaAki** [reply]
谢谢！
